test/fixtures/schema/array-fixed.json

```diff
@@ -182,7 +182,7 @@
                                   "content": "application/schema+json"
                                 }
                               },
-                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"tags\": {\n      \"type\": \"array\",\n      \"minItems\": 2,\n      \"items\": [\n        {\n          \"type\": \"string\",\n          \"enum\": [\n            \"hello\"\n          ]\n        },\n        {\n          \"type\": \"number\",\n          \"enum\": [\n            42\n          ]\n        }\n      ],\n      \"additionalItems\": false\n    }\n  }\n}"
+                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"tags\": {\n      \"type\": \"array\",\n      \"minItems\": 2,\n      \"items\": [\n        {\n          \"enum\": [\n            \"hello\"\n          ]\n        },\n        {\n          \"enum\": [\n            42\n          ]\n        }\n      ],\n      \"additionalItems\": false\n    }\n  }\n}"
```

Comment:
I gave it a go, I don't think this will handle samples but it does work for this case at least. Not sure if it's worth doing anything like this though.

```diff
diff --git a/src/refract/JsonSchema.cc b/src/refract/JsonSchema.cc
index c2cb5411..29667c2c 100644
--- a/src/refract/JsonSchema.cc
+++ b/src/refract/JsonSchema.cc
@@ -375,6 +375,12 @@ namespace
             addItems(schema, so::Object{ so::from_list{}, std::make_pair("anyOf", std::move(items)) });

         } else if (options.test(FIXED_FLAG)) { // tuple of N constants/types
+            if (sizeOf(e, true) == cardinal{ 1 }) {
+                so::Array enm{};
+                so::emplace_unique(enm, generateJsonValue(e));
+                addEnum(s, std::move(enm));
+                return s;
+            }

             so::Array items{};
             if (!e.empty())
```